### PR TITLE
[ch10046] output mergedProjectData, discard projectData.S, projectBinaryData.B

### PR DESCRIPTION
**Pull request by dcsobral**

### CODE REVIEW

File: ETL/dynamodb-etl.sh [hunk 1]
```diff
@@ -91,25 +91,33 @@ while [[ $# -gt 0 && $1 == -* ]]; do
...
 	| \$line
-	| if getpath(${BINARY_JQ_PATH}) then setpath(${BINARY_JQ_PATH}; \$uncompressed) else . end
-	| if getpath(${STRING_JQ_PATH}) then setpath(${STRING_JQ_PATH}; \$literal) else . end
+	| setpath(${MERGED_JQ_PATH};
+	    if getpath(${BINARY_JQ_PATH}) then \$uncompressed
+		else if getpath(${STRING_JQ_PATH}) then \$literal
+		else null end
+        end
+	  )
+    | del(${BINARY_PATH})
```
Comment (nicflores) [inline ETL/dynamodb-etl.sh]: should this be moved two spaces to the right?

Comment (dcsobral) [reply ETL/dynamodb-etl.sh]: I'll check. I see there's mixed tabs and spaces there. :(

Comment (dcsobral) [reply ETL/dynamodb-etl.sh]: It should be fixed now. The alignment was correct for 4 spaces per tab, but some lines were using spaces instead.

Comment (dcsobral) [reply ETL/dynamodb-etl.sh]: I take that back. Someone some lines weren't changed.

Comment (dcsobral) [reply ETL/dynamodb-etl.sh]: Uh, no, things are fine. Funny, the diff shows spaces, but I verified they are tabs locally, and there's nothing uncommitted, and everything was pushed.

Comment (dcsobral) [reply ETL/dynamodb-etl.sh]: And now the diff is right. I must have hit an old version.

File: ETL/dynamodb-etl.sh [hunk 2]
```diff
@@ -21,7 +21,7 @@ fi
 
 usage() {
 	cat >&2 <<-USAGE
-		Usage: $0 [-?|-h|--help] [<binary_path> <string_path>]
+		Usage: $0 [-?|-h|--help] [<binary_path> <string_path> <merged_path]
```
Comment (nicflores) [inline ETL/dynamodb-etl.sh]: should this be `<merged_path>`?

File: ETL/dynamodb-etl.sh [hunk 3]
```diff
@@ -91,25 +91,33 @@ while [[ $# -gt 0 && $1 == -* ]]; do
...
+MERGED_JQ_PATH="$(jq -n -c "path($MERGED_PATH)")"
 # False positive on -d 'EOL':
 # shellcheck disable=SC2034
 read -r -d 'EOL' OUTPUT_QUERY <<-QUERY
 	. as \$uncompressed
 	| (\$line | ${STRING_DEFAULT_QUERY} | fromjson) as \$literal
 	| \$line
-	| if getpath(${BINARY_JQ_PATH}) then setpath(${BINARY_JQ_PATH}; \$uncompressed) else . end
-	| if getpath(${STRING_JQ_PATH}) then setpath(${STRING_JQ_PATH}; \$literal) else . end
+	| setpath(${MERGED_JQ_PATH};
```
Comment (nicflores) [inline ETL/dynamodb-etl.sh:113]: Sorry, but I don't see where functions `setpath()`, `getpath()`, `path()`, and `del()` are coming from? Are these bash function?

File: ETL/dynamodb-etl.sh (hunk 2 above)
Comment (dcsobral) [reply ETL/dynamodb-etl.sh]: Yes.

File: ETL/dynamodb-etl.sh (hunk 3 above)
Comment (dcsobral) [reply ETL/dynamodb-etl.sh:113]: These are jq functions. This text is the jq script.

Comment (nicflores) [reply ETL/dynamodb-etl.sh:113]: Ah! Thanks for that point.

**Review by nicflores — APPROVED** [review]

**Pull request merged**